layouts/shortcodes/pages/release.html

```diff
@@ -1,104 +1,99 @@
-<div class="jakarta-download-delivers container">
-  <h2 class="heading-underline heading-underline-extended text-center">Jakarta EE 8 Delivers</h2>
-  
-  <div class="row download-highlights margin-top-40">
-    <div class="col-sm-12 download-highlights-item">
-      <span class="featured-circle-outline"><img width="50" class="icon margin-top-10" src="images/list.svg"></span>
-      <h3>Full Compatibility with Java<sup>TM</sup>&nbsp;EE 8</h3>
-      <p>Fully compatible with the Java EE 8 specifications and Technology Compatibility Kits (TCKs)</p>
-    </div>
-    <div class="col-sm-12 download-highlights-item">
-      <span class="featured-circle-outline"><img width="50" class="icon margin-top-10" src="images/design-1.svg"></span>
-      <h3>Open Specifications</h3>
-      <p>Leveraging the <a href="/about/jesp/">Jakarta EE Specification Process</a>, an open source specification process that enables code-first development</p>
+<div class="features-and-benefits container">
+  <h2 class="text-center text-blue fw-600 uppercase"><strong>jakarta ee 9</strong></h2>
+  <h2 class="text-center text-primary fw-600 margin-top-0 font-exo uppercase"><em>features and benefits</em></h2>
+  <div class="row margin-top-40">
+    <div class="col-md-8 col-sm-8">
+      <div class="feature-box">
+        <div class="polygons polygons-right vertical-align"><img width="50" class="icon" src="images/release-9/icon1.svg" alt="lower entry barriers"></div>
+        <h3 class="fw-400 uppercase">lower entry <br> barriers</h3>
+        <hr />
+        <p>Jakarta EE 9 lowers the barrier of entry to new vendors and implementations to achieve compatibility</p>
+      </div>
     </div>
-    
-  </div>
-  <div class="row download-highlights margin-top-30">
-    <div class="col-sm-12 download-highlights-item">
-      <span class="featured-circle-outline"><img width="50" class="icon margin-top-15" src="images/document.svg"></span>
-      <h3>Open Source TCK License and Process</h3>
-      <p>An open process for certifying products as Compatible Implementations of Jakarta EE Profiles using TCKs licensed under the Eclipse TCK License</p>
+    <div class="col-md-8 col-sm-8">
+      <div class="feature-box">
+        <div class="polygons polygons-right vertical-align"><img width="50" class="icon" src="images/release-9/icon2.svg" alt="foundation for innovation"></div>
```

Comment:
Alts should describe what is shown, not where something is

Reply:
You mean the content under the title? like:
`Jakarta EE 9 lowers the barrier of entry to new vendors and implementations to achieve compatibility`

Reply:
Used "aria-hidden" instead. Since no point to describe the icon.